### PR TITLE
2019 Reading Review Post

### PR DESCRIPTION
**Pull request by benhamill**
Oh wow many books.

### CODE REVIEW

File: _posts/2020-01-06-2019-reading-in-review.md [hunk 1]
```diff
@@ -0,0 +1,501 @@
...
+read in trade paperback form, or stuff I read with my kids, which is mostly very
+short but has begun to include some chapter books. Many of those things have
+been lovely and totally worth recommending or talking about, but I figure I have
+to limit scope somehow and I didn't happen to think to put those into my
+journal.
+
+
+## The Numbers
+
+Let's do some stats, shall we? Obviously I have to excluse some stuff from the
```
Comment (alexj) [inline _posts/2020-01-06-2019-reading-in-review.md]: "excluse" → "exclude"

File: _posts/2020-01-06-2019-reading-in-review.md [hunk 2]
```diff
@@ -0,0 +1,501 @@
...
+duration stats on the basis of not knowing when I read it.
+
+**Number of Things Read:** 47
+
+**Reading Time** (including weekends, during which I generally don't read):
```
Comment (alexj) [inline _posts/2020-01-06-2019-reading-in-review.md]: Is this the number of days you spent reading each book?

File: _posts/2020-01-06-2019-reading-in-review.md [hunk 3]
```diff
@@ -0,0 +1,501 @@
...
+* Maximum: 21 days (_Jacobin 33_ and _The Prey of Gods_)
+* [Median](https://en.wikipedia.org/wiki/Median): 8 days (_A Conjuring of Light_ and _Jacobin 34_)
+* Minimum: 0 days (_Artificial Condition_, _Exit Strategy_, _Capitalism & the
```
Comment (alexj) [inline _posts/2020-01-06-2019-reading-in-review.md]: 0 days implies that you didn't read these to me. If you're indicating less than a day, something other than zero seems like a better path.

File: _posts/2020-01-06-2019-reading-in-review.md [hunk 4]
```diff
@@ -0,0 +1,501 @@
...
+  State_)
+* [Mode](https://en.wikipedia.org/wiki/Mode_(statistics)): 13 days (_Washington Black_, _The Body Keeps
+  the Score_, _Broken Stars_ and _The City of Brass_)
+
+
+## Things I Read in 2019
+
+The format is the title in italics, then the author's name in normal text (for
```
Comment (alexj) [inline _posts/2020-01-06-2019-reading-in-review.md]: I'm not sure this paragraph is all that useful to the reader

File: _posts/2020-01-06-2019-reading-in-review.md [hunk 5]
```diff
@@ -0,0 +1,501 @@
...
+one increasingly causes the other over time. I think they also do a good job of
+making the point that there is only so far we can get to achieve justice for all
+without overthrowing capitalism and, thus, the limits we're generally under if
+we constrain our efforts to electoral politics.
+
+
+### _Star Wars: Last Shot_ by Daniel José Older (2019-03-04)
+
+I think this is a story about Lando and Han? There's a non-binary character in
+it who was pretty cool, I think? And maybe it hopped between eras of, like,
```
Comment (alexj) [inline _posts/2020-01-06-2019-reading-in-review.md]: Do you think it's pretty cool that there's a non-binary character or do you think the character is non-binary?

File: _posts/2020-01-06-2019-reading-in-review.md [hunk 6]
```diff
@@ -0,0 +1,501 @@
...
+
+
+### _After On_ by Rob Reid (???)
+
+I remember liking this book for the most part and a lot of the ideas were quite
+interesting, but it had a fondness and adulation for Silicon Valley start-up
+culture that I really couldn't stand and the resolution to the main conflict
+boiled down to what a lot of authoritarian fiction (like super hero stories)
+does: A small number of smart/good/brave/powerful/special individuals should
+stand between everyone else and the problem. Bleh.
```
Comment (h3h) [inline _posts/2020-01-06-2019-reading-in-review.md]: Aside: Do you have recommendations for anything where a federated/anarchic solution is plausibly implemented to handle such issues?

**Pull request merged**